data/filedefs/filetypes.cobol

```diff
@@ -12,11 +12,11 @@ string=string_1
 character=character
 operator=operator
 identifier=identifier_1
-quotedidentifier=identifier_2
+preprocessor=preprocessor
```

Comment:
`word3` looks missing

Reply:
Yes, I haven't mapped it since there's only one keyword group anyway. But I haven't checked what "policy" we use with other lexers.

Reply:
There seems to be `keyword_3` so I could map it easily to this one. I'll submit a fix.

Reply:
…which was mapped ad `other`, but I guess using `word3=keyword_3` would be better